AquaNet/src/App.svelte

```diff
@@ -1,12 +1,31 @@
 <script lang="ts">
-  import { Router, Route } from "svelte-routing";
+  import {Router, Route} from "svelte-routing";
   import Welcome from "./pages/Welcome.svelte";
   import MaimaiRating from "./pages/MaimaiRating.svelte";
   import UserHome from "./pages/UserHome.svelte";
-  import Icon from '@iconify/svelte';
   import Home from "./pages/Home.svelte";
   import Ranking from "./pages/Ranking.svelte";
-  import { USER } from "./libs/sdk";
+  import {DATA, USER} from "./libs/sdk";
+  import {onMount} from "svelte";
+  import {fetchUserDetails, type UserDetails} from "./libs/FetchGameUserInfo";
+  import {DATA_HOST} from "./libs/config";
+  import {pfpNotFound} from "./libs/ui";
+  import type {GameName} from "./libs/scoring";
+
+  let userIcon: string | null;
+  let game = "mai2"
+  let username = "waitApiUpdate" //wait api update
+  let userImg:string
+  onMount(() => {
+    fetchUserDetails(username, <GameName>game).then((details) => {
+      userIcon = details?.user?.iconId?.toString().padStart(6, "0") ?? null;
+      userImg = DATA_HOST + "/d/" + game + "/assetbundle/icon/" + userIcon + ".png"
+      console.log("app", userImg)
+    }).catch();
```

Comment:
NEVER EVER catch with no statement. Change this to:
`.catch(console.error)`
Or if you already knows that in some condition it will throw exception, handle it.